### PR TITLE
feat: Add _messageHash to ItemSlotsConsumed event

### PR DESCRIPTION
**Pull request by fzavalia**
^ Title

### CODE REVIEW

File: test/helpers/thirdPartyRegistry.js [hunk 1]
```diff
@@ -81,3 +82,43 @@ export async function getSignature(
     v,
   }
 }
+
+export async function getMessageHash(
```
Comment (nachomazzara) [inline test/helpers/thirdPartyRegistry.js:86]: Why do we need to make it to return a promise?

Comment (nachomazzara) [reply test/helpers/thirdPartyRegistry.js:86]: I would receive the `chainId` by parameter

Comment (fzavalia) [reply test/helpers/thirdPartyRegistry.js:86]: I'm doing the same as `getSignature` does

File: contracts/registries/ThirdPartyRegistry.sol [hunk 2]
```diff
@@ -106,7 +106,7 @@ contract ThirdPartyRegistry is OwnableInitializable, NativeMetaTransaction, Init
     event ItemAdded(string _thirdPartyId, string _itemId, string _metadata, bool _value, address _caller);
     event ItemUpdated(string _thirdPartyId, string _itemId, string _metadata, address _caller);
     event ItemReviewed(string _thirdPartyId, string _itemId, string _metadata, string _contentHash, bool _value, address _caller);
-    event ItemSlotsConsumed(string _thirdPartyId, uint256 _qty, address indexed _signer, address indexed _sender);
+    event ItemSlotsConsumed(string _thirdPartyId, uint256 _qty, address indexed _signer, address indexed _sender, bytes32 _messageHash);
```
[the remark this answers is not in the document]
Comment (fzavalia) [reply contracts/registries/ThirdPartyRegistry.sol]: Done

**Review by nachomazzara — APPROVED** [review]

**Pull request merged**